### DISCUSSION TITLE
Getting error when upgrading from version 4.1 to 4.3

### DISCUSSION THREAD

**Discussion by PKASHYAP-BEC** [iTwin/itwinjs-core#6290] [General]
getting error "TypeError: nativeDb.closeIModel is not a function" when trying to createNewImodel method 
![image](https://github.com/iTwin/itwinjs-core/assets/60220061/cd375e5a-c3f9-479a-85d0-9302d2fbe8cf)

https://github.com/iTwin/imodels-clients/blob/main/itwin-platform-access/imodels-access-backend/package.json

When we can expect imodel-access-backend compatible with 4.2 & 4.3 versions of other packages

**Comment by pmconne** [comment]
@austeja-bentley

**Comment by pmconne** [comment]
This will [be fixed](https://github.com/iTwin/itwinjs-core/pull/6277) in @itwin/core-backend 4.3.1.
Many apologies for the inconvenience. We'll get 4.3.1 released as soon as possible.

**Reply by aruniverse** [reply]
4.3.1 has been released, and the issue should be resolved now. Apologies for any inconvenience. Please let us know if you run into any other issues

**Reply by rkumarsbechtel** [reply]
Hi  @aruniverse , @pmconne ,

We checked after upgrading packages to version 4.3.1, but issue is still not resolved because the problem is there with @itwin/imodels-access-backend package.

I am expecting this package to get updated along with other core packages.

Thanks,

**Reply by austeja-bentley** [reply]
We fixed the `@itwin/imodels-access-backend`, please try the version 4.1.3 and let us know if the issue persists.

**Reply by Rock2000** [reply]
I updated my app, which uses itwin 4.4.0-dev.10, but I'm still getting this error.

**Reply by austeja-bentley** [reply]
@Rock2000, what version of `@itwin/imodels-access-backend` package are you using?

**Reply by pmconne** [reply]
He's using 4.1.4 and according to his comments in the [other discussion](https://teams.microsoft.com/l/message/19:00f67b80776d4705942f17cd4eef6591@thread.skype/1696451027857?tenantId=067e9632-ea4c-4ed9-9e6d-e294956e284b&groupId=2fc3ba7e-a05a-4cc0-9411-e9d2d2652ec1&parentMessageId=1696451027857&teamName=iTwin.js&channelName=Questions&createdTime=1696451027857) about this, he's not getting this same error, he's getting "Expected signal to be an instanceof AbortSignal" from node-fetch.
